NUnit.proj

```diff
@@ -501,18 +498,9 @@
     <Message Text="* Creating the binary $(Configuration) package as $(PackageNameCF)$(ConfigSuffix).zip" />
     <Message Text="******************************************************************" />
 
-    <CallTarget Targets="_RemoveGitDirFromImage" />
-
-    <Exec Command='git init "$(CurrentImageDir)"' />
-    <Exec WorkingDirectory='$(CurrentImageDir)' Command='git add *.txt nunit.ico' />
-    <Exec WorkingDirectory='$(CurrentImageDir)' Command='git add bin\netcf-3.5\*' />
-    <Exec WorkingDirectory='$(CurrentImageDir)' Command='git config user.name "temp"' />
-    <Exec WorkingDirectory='$(CurrentImageDir)' Command='git config user.email "temp@temp.com"' />
-    <Exec WorkingDirectory='$(CurrentImageDir)' Command='git commit -m "Export"' />
-    <Exec WorkingDirectory='$(CurrentImageDir)'
-      Command='git archive -o "$(ProjectPackageDir)\$(PackageNameCF)$(ConfigSuffix).zip" HEAD'></Exec>
+    <Zip Files="@(CFBinZipFiles)" 
```

Comment:
Above in the `PackageBinaries` target you set ` WorkingDirectory="$(CurrentImageDir)"` is that not necessary here?